File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/EnemyCloudDragonKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class EnemyCloudDragonKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playEnemyCloudDragonKillAnimation() {
        new AnimationTester().testAnimation(new EnemyCloudDragonKillAnimation());
    }
}

File: test/hud/animations/EnemyCloudDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {EnemyCloudDragonKillAnimation} from '../../../src/hud/animations/EnemyCloudDragonKillAnimation.js';

/**
 * Integration test for enemy cloud dragon kill animation on actual hardware.
 */
describe('EnemyCloudDragonKillAnimation Integration', () => {
    test('playEnemyCloudDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new EnemyCloudDragonKillAnimation());
    }, 30000);
});
